### PR TITLE
virt-manager updated to version 2 - GUI changes

### PR DESCRIPTION
**Pull request by tbazant**
### Description

virtual machine manager got upgraded to version 2, resulting in several changes to procedures and related GUI

### Are there any relevant issues/feature requests?

* jsc#DOCTEAM-40
* jsc#SLE-4572

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ----------
| [x] 15 SP3  | *(no backport necessary)*
| [x] 15 SP2  | [ ]
| [x] 15 SP1  | [ ]
| [ ] 15 SP0  | [ ]

| Code 12     | Backport Done
| ----------  | ----------
| [ ] 12 SP5  | [ ]
| [ ] 12 SP4  | [ ]
| [ ] 12 SP3  | [ ]
| [ ] 12 SP2  | [ ]

### CODE REVIEW

**Review by lproven — CHANGES_REQUESTED** [review]
No direc

File: xml/libvirt_guest_installation.xml [hunk 1]
```diff
@@ -38,13 +38,9 @@
...
-   The <guimenu>New VM</guimenu> wizard helps you through the steps required to
-   create a virtual machine and install its operating system. There are two
-   ways to start it: Within &vmm;, either click <guimenu>Create New Virtual
-   Machine</guimenu> or choose <menuchoice> <guimenu>File</guimenu>
-   <guimenu>New Virtual Machine</guimenu> </menuchoice>. Alternatively, start
-   &yast; and choose <menuchoice><guimenu>Virtualization</guimenu>
-   <guimenu>Create Virtual Machines for Xen and KVM</guimenu></menuchoice>.
+   The <guimenu>New VM</guimenu> wizard helps you through the steps required to create a virtual machine and install its operating system.
+   To start it, open the &vmm; and select <menuchoice><guimenu>File</guimenu> <guimenu>New Virtual Machine</guimenu></menuchoice>.
+   Alternatively, start &yast; and select <menuchoice><guimenu>Virtualization</guimenu> <guimenu>Create Virtual Machines</guimenu></menuchoice>.
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/libvirt_guest_installation.xml:43]: it is on purpose but i can cancel that experiment...

File: xml/libvirt_host.xml [hunk 2]
```diff
@@ -475,68 +456,30 @@ virbr_test  1 PVID Egress Untagged
...
-      <step>
-       <para>
-        If you did not specify an IPv6 network address space definition
-        earlier, you can enable IPv6 internal routing between virtual machines.
-       </para>
-      </step>
-      <step performance="optional">
-       <para>
-        Optionally, change the DNS domain name.
+        If you need a different domain name than the name of the virtual network, select <guimenu>Custom</guimenu> under <guimenu>DNS domain name</guimenu> and type it.
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/libvirt_host.xml:482]: yep

File: xml/libvirt_host.xml [hunk 3]
```diff
@@ -1708,7 +1616,12 @@ done</screen>
           </listitem>
           <listitem>
            <para>
-            <guimenu>Source Path</guimenu>: The iSCSI target name (IQN).
+            <guimenu>Source IQN</guimenu>: The iSCSI target name (IQN).
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            <guimenu>Initiator IQN</guimenu>: The iSCSI initiator name (IQN).
```
Comment (tbazant) [inline xml/libvirt_host.xml:1627]:
```suggestion
            <guimenu>Source IQN</guimenu>: The iSCSI target name (iSCSI Qualified Name).
           </para>
          </listitem>
          <listitem>
           <para>
            <guimenu>Initiator IQN</guimenu>: The iSCSI initiator name (iSCSI Qualified Name).
```

**Review by tbazant — COMMENTED** [review]
looks good

**Review by lproven — APPROVED** [review]

**Review by lproven — APPROVED** [review]

File: xml/libvirt_host.xml [hunk 4]
```diff
@@ -1708,7 +1619,12 @@ done</screen>
           </listitem>
           <listitem>
            <para>
-            <guimenu>Source Path</guimenu>: The iSCSI target name (IQN).
+            <guimenu>Source IQN</guimenu>: The iSCSI target name (iSCSI Qualified Name).
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            <guimenu>Initiator IQN</guimenu>: The iSCSI initiator name (iSCSI Qualified Name).
```
Comment (lproven) [inline xml/libvirt_host.xml:1627]: Nice. :-) I think it is only necessary the 1st time, though... unless you disagree.

**Review by lproven — APPROVED** [review]
With one query, but if you are in a rush, then go for it.

File: xml/libvirt_guest_installation.xml [hunk 5]
```diff
@@ -89,14 +85,12 @@
       </listitem>
      </varlistentry>
      <varlistentry>
-      <term><guimenu>Network installation (HTTP, FTP, or NFS)</guimenu>
+      <term><guimenu>Network install (HTTP, HTTPS, or FTP)</guimenu>
```
Comment (lproven) [inline xml/libvirt_guest_installation.xml:88]: I preferred "installation", to be honest.

CD-ROM or DVD-ROM? Isn't DVD more likely?

Comment (tbazant) [reply xml/libvirt_guest_installation.xml:88]: Me too but that's what the GUI says now...

File: xml/libvirt_host.xml (hunk 4 above)
Comment (tbazant) [inline xml/libvirt_host.xml:1627]:
```suggestion
            <guimenu>Initiator IQN</guimenu>: The iSCSI initiator name.
```

**Review by tbazant — COMMENTED** [review]
minor update

File: xml/libvirt_guest_installation.xml [hunk 6]
```diff
@@ -109,6 +103,7 @@
        </para>
       </listitem>
      </varlistentry>
+     <!-- 2021-04-13 tbazant, Network boot no longer there...
```
Comment (aginies) [inline xml/libvirt_guest_installation.xml:106]: its still possible in "Manual install" which is more generic

**Review by aginies — APPROVED** [review]
Thanks for the changes

**Pull request merged**